docs/user-guide/semdedup.rst

```diff
@@ -259,6 +242,7 @@ Key parameters in the configuration file include:
 - ``n_clusters``: Number of clusters for k-means clustering.
 - ``eps_to_extract``: Deduplication threshold. Higher values result in more aggressive deduplication.
 - ``which_to_keep``: Strategy for choosing which duplicate to keep ("hard" or "soft").
+- ``sim_metric``: Similarity metric to use to rank within cluster. The order is determined by which_to_keep.
```

Comment:
Nit: "The order is determined by which_to_keep" is kind of confusing for me to understand.